cmd/manager/main.go

```diff
@@ -65,10 +66,11 @@ func main() {
 
 	printVersion()
 
-	namespace, err := k8sutil.GetWatchNamespace()
+	watchNamespace, err := k8sutil.GetWatchNamespace()
```

Comment:
This comes from the env var `WATCH_NAMESPACE`.  That is set previously this was set to `metadata.annotations.olm.targetNamespaces` which was `""` and then ignored.  The operator was always using `""` which watches all namespaces.

`WATCH_NAMESPACE`:
- Leaving it as `""` will continue with the current behavior of monitoring all namespaces.
- Setting it to a single namespace will monitor that namespace.
- Setting it to comma separated namespaces in the CSV, e.g. `redhat-ods-operator,redhat-ods-applications,redhat-ods-monitoring,rhods-notebooks`., will monitor those namespaces.